cli/src/format.rs

```diff
@@ -102,54 +102,50 @@ impl Format for ListGroupMembersResponse {
     }
 }
 
+/// Write object fields
+fn write_fields<W: Write>(fields: &[(&str, &str)], writer: &mut W) -> std::io::Result<()> {
+    let max_label_width = fields.iter().map(|f| f.0.len()).max().unwrap_or(0);
+
+    for field in fields {
+        writeln!(writer, "  {}  {}", style(leftpad(field.0, max_label_width)).blue(), field.1)?;
+    }
+    Ok(())
+}
+
 impl Format for Vec<JobDescriptor> {
     fn pretty<W: Write>(&self, writer: &mut W) {
         let _ = writeln!(writer, "Last {} runs\n", self.len());
 
-        let mut jobs = String::new();
-        for (i, job) in self.iter().enumerate() {
-            let mut state = style("PASS").green().to_string();
-            let score = format!("{:>3}", (job.score * 100.0) as u32);
-            let mut colored_score = style(&score).green().to_string();
-            let project_name = print::truncate(&job.project, 39);
-            let colored_project_name = style(project_name).white().bold().to_string();
-
-            if job.num_incomplete > 0 {
-                colored_score = format!("{}", style(&score).yellow());
-                state = format!("{}", style("INCOMPLETE").yellow());
+        for job in self {
+            let status = if job.num_incomplete > 0 {
+                style("INCOMPLETE").yellow().to_string()
```

Comment:
In the CI integrations, there is another status to indicate `INCOMPLETE WITH FAILURE`. Does it make sense to add that here? It could also be colored red.

Reply:
Good question... I don't like `INCOMPLETE WITH FAILURE` simply because it is too long. I'd prefer:

1. Keep `INCOMPLETE` but use red text
1. Just use `FAIL` because that will be the final result anyways

Or maybe I'll think of another option before I come back to this tomorrow...

Reply:
:+1: I've updated the language to match our other integrations.